assets/base/i18n/en.json

```diff
@@ -377,6 +376,7 @@
   "mobile.calls_limit_msg": "The maximum number of participants per call is {maxParticipants}. Contact your System Admin to increase the limit.",
   "mobile.calls_limit_reached": "Participant limit reached",
   "mobile.calls_lower_hand": "Lower hand",
+  "mobile.calls_mic_error": "To participate, open Settings to grant Mattermost access to your microphone, then rejoin the call.",
```

Comment:
```suggestion
  "mobile.calls_mic_error": "To participate, open Settings to grant Mattermost access to your microphone. Then leave and rejoin the call.",
```

Reply:
@cpoile, I'm still not 100% sure of this - I didn't realize how small the screen real estate is.

Is the user able to connect to the call, then get this message? If so, we could cut the second sentence down to "Then join the call."

Reply:
I figured out how to reset the mic permissions, so we don't need this different string. Thanks Justine!